.github/workflows/test.yml

```diff
@@ -0,0 +1,69 @@
+name: Test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.11.3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.1.0
+
+      - name: Lint chart
+        run: ct lint --charts chart
+
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.11.3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.4.0
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.4.0
+
+      - name: Set up kubectl
+        uses: azure/setup-kubectl@v3
+
+      - name: Install Helm chart
+        run: |
+          helm dependency build ./chart
+          helm repo add flanksource https://flanksource.github.io/charts
+          helm repo add ory https://k8s.ory.sh/helm/charts
+          ct install --charts ./chart --namespace default
+
+      - name: Test deploying topology
+        run: |
+          kubectl apply -f topologies/mission-control.yaml
+          kubectl get topology mission-control -o yaml
+          kubectl wait --for=condition=ready --timeout=5m topology mission-control
```

Comment:
Should we also execute a shell test via curl, like for testing some data in incidents or peoples table etc. ?

Reply:
Sure. If you suggest an endpoint to test I can add that